.github/workflows/test.yaml

```
@@ -16,6 +16,26 @@ jobs:
           otp-version: "27.2"
           gleam-version: "1.9.1"
           rebar3-version: "3"
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)